features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# Description
+
+This document describes how to update the mbed OS LWIP stack. The mbed OS LWIP stack is a copy of the LWIP master repository. Stack is located in `FEATURE_LWIP/lwip-interface/lwip` directory.
```

Comment:
Yesterday, Arm rolled out new branding guidelines. To reflect these changes please change "update the mbed OS" to update the Arm Mbed OS". In the second sentence, please change "mbed" to "Mbed".